easy-to-use-api/klips-warning-api/src/components/GetData/index.ts

```diff
@@ -0,0 +1,104 @@
+// @ts-ignore
```

Comment:
Can this be imported without the @ts-ignore? Maybe we need to do some adjustments in the tsconfig.json

Reply:
Will be revisited in a follow-up.